test/unittests/eof_validation_test.cpp

```diff
@@ -733,6 +734,75 @@ TEST(eof_validation, callf_invalid_code_section_index)
         EOFValidationError::invalid_code_section_index);
 }
 
+TEST(eof_validation, callf_stack_overflow)
+{
+    {
+        auto code = eof1_bytecode(
+            512 * push(1) + OP_CALLF + bytecode{"0x0000"_hex} + 510 * OP_POP + OP_RETURN, 512);
```

Comment:
```suggestion
            512 * push(1) + OP_CALLF + "0000" + 510 * OP_POP + OP_RETURN, 512);
```
`bytecode` can consume strings and will interpret them as hex.